assets/js/modules/analytics/components/common/UseUAandGA4SnippetSwitches.js

```diff
@@ -87,14 +103,17 @@ export default function UseUAandGA4SnippetSwitches() {
 						disabled={ ! canUseUASnippet }
 					/>
 				</div>
-				<div className="googlesitekit-settings-module__inline-item">
-					<Switch
-						label={ __( 'Place Google Analytics 4 code', 'google-site-kit' ) }
-						checked={ useGA4Snippet }
-						onClick={ onGA4Change }
-						hideLabel={ false }
-					/>
-				</div>
+
+				{ useGA4Snippet !== undefined && (
```

Comment:
Nit-pick: This is technically correct of course, but it reads a bit confusing. We mostly use `undefined` checks for some sort of loading state check, but here that is different. I think it would be a more self-explanatory approach to define a variable above like `const showGA4Toggle = isGA4Enabled && select( CORE_MODULES ).isModuleConnected( 'analytics-4' ) === select( CORE_MODULES ).isModuleConnected( 'analytics' )`

Reply:
Updated.